lib/cfn-nag/custom_rules/EC2NetworkAclEntryDuplicateRule.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class EC2NetworkAclEntryDuplicateRule < BaseRule
+  def rule_text
+    'A NetworkACL\'s rule numbers cannot be repeated unless one is egress and one is ingress.'
+  end
+
+  def rule_type
+    Violation::FAILING_VIOLATION
+  end
+
+  def rule_id
+    'F79'
+  end
+
+  def audit_impl(cfn_model)
+    nacl_violating_egress_entries = []
+    nacl_violating_ingress_entries = []
+    cfn_model.resources_by_type('AWS::EC2::NetworkAcl').select do |nacl|
```

Comment:
do you mean each here instead of select?

Reply:
Updated